src/components/10-atoms/icon/index.js

```diff
@@ -55,26 +55,25 @@ class AXAIcon extends LitElement {
   static get properties() {
     return {
       icon: { type: String },
-      size: { type: String, reflect: true },
+      size: { type: String, reflect: true, defaultValue: 'medium' },
       _loadedSvg: { type: String },
     };
   }
 
   constructor() {
     super();
     applyDefaults(this);
+    this.size = 'medium';
```

Comment:
This is redundant to line 58. I would anyway delete both lines and add a default parameter to the render function.

Reply:
Is it needed that you declare it as 'medium'? I don't see where that value is used in here.

Reply:
redundant: there are cases that the class is instanciated without the default values of the props (marco told me that).

medium: this prop is reflected. So, the user can see the attribute on his dom node.

Reply:
But shouldn't we then add the default value only in the constructor? Or do we need both?

Reply:
I saw that only `axa-carousel` is setting these values at the constructor. Maybe markos issue is fixed by `applyDefaults()`. I will remove it from constructor.